stake/src/rate.rs

```diff
@@ -0,0 +1,81 @@
+use penumbra_crypto::rdsa::{SpendAuth, VerificationKey};
+use penumbra_proto::{stake as pb, Protobuf};
```

Comment:
Importing the corresponding proto module `as pb` makes it easy to consistently name both the domain types `Foo` and the proto types `pb::Foo`; elsewhere in the code we're a bit less consistent.